tests/Template/FunctionTemplateTest.php

```diff
@@ -291,7 +291,7 @@ function splat_proof(array $arr, array $arr2) {
 
                     $a = splat_proof(...$foo);',
                 'assertions' => [
-                    '$a' => 'array<int, int>',
+                    '$a' => 'array<int<0, 2>, int>',
```

Comment:
I feel like int ranges where the `max` is 2 should render as `0|1|2`

Reply:
Well, that's not just a display preference, it will have different effects on things afterwards (union of 3 literal ints are not always handled the same way as a range)

Reply:
yeah, I get it, it was a bad feeling!